app/views/events/index.slim

```diff
@@ -2,10 +2,12 @@ h2 Events
 -@events.each do |event|
   = link_to event.name, event_path(event.id)
   ul
-    li 
+    li
       b = "On #{Date::DAYNAMES[event.start_date.wday]}"
     li = event.starting_date
-    li = event.ending_date
       li.fa.fa-user = "#{event.attendees.size}"
+    -unless current_user.nil?
```

Comment:
Yes i can.
I'm going with your first recommendation.
Basically this check is useless.